scripts/publish.sh

```diff
@@ -37,16 +37,17 @@ function processCommandLineArgs {
 
 function release {
   BUMP=$(npx -p conventional-changelog-angular -p conventional-recommended-bump -c 'conventional-recommended-bump -p angular')
-  VERSION=$(npm --no-git-tag-version version ${1:-$BUMP})
+  VERSION=$(yarn version --no-git-tag-version --new-version ${1:-$BUMP})
```

Comment:
cool, `--new-version` does accept `major|minor|patch`, I thought it was only for providing a specific version to use.